app/jsx/components/RefineComp.jsx

```diff
@@ -0,0 +1,143 @@
+// ##### Refine Component ##### //
+
+import React from 'react'
+import Breakpoints from '../../js/breakpoints.json'
+import PubYearComp from '../components/PubYearComp.jsx'
+
+class RefineComp extends React.Component {
+  state={refineActive: false, drawerOpen: false}
+
+  componentWillMount() {
+    if (matchMedia) {
+      this.mq = matchMedia("(min-width:"+Breakpoints.screen1+")")
+      this.mq.addListener(this.widthChange)
```

Comment:
Seems like we need a corresponding removeListener (or whatever matchMedia's API is) in componentDidUnmount